src/main/resources/application.yaml

```diff
@@ -34,7 +34,7 @@ idam:
   s2s-auth:
     url: ${SERVICE_AUTH_PROVIDER_URL:http://localhost:4502}
     secret: ${ET_SYA_S2S_SECRET:AAAAAAAAAAAAAAAC}
-    microservice: ${MICRO_SERVICE:et_sya_api}
+    microservice: ${MICRO_SERVICE:ccd_gw}
```

Comment:
Just return this to et_sya_api for now. I know it needs to be like the above to work locally but we need to update ecm-ccd-docker to fix that.

Reply:
Changed back to et_sya_api